src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -50,13 +53,33 @@ const transformJson = async (json) => {
   }
 };
 
+const getAdditionalData = async (pageData) => {
+  const pageType = getPageType(pageData);
+
+  switch (pageType) {
+    case STORY_PAGE:
+      const { json: secondaryColumData } = await fetchPageData(
+        '/mundo/testdata',
+      );
+      return secondaryColumData;
+    default:
+      return;
+  }
+};
```

Comment:
Are we expecting to load additional data for other pageTypes? If yes, should we then pass the endpoint `/mundo/testdata` as an argument.

Reply:
we might at some point, although not in this PR.

As this is just a draft PR to show. POC this wont be getting merged. So this current hardcoded path would be replaced with a function that generates the urls based on the service

Reply:
Understood. Thanks.